dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
I think it should be scoped.  Its nice that its a target since you don't need to do a full build to get the answer.  FWIW we already have a GetPackageIdentity target in pkgproj but it might be a bit heavy since it calculates things today.